bosshud/ze_lotr_isengard_csgo1.txt

```diff
@@ -0,0 +1,56 @@
+"math_counter"
+{
+	"config"
+	{
+		"HitMarkerOnly"				"1"

```

Comment:
This shouldn't be enabled, since the inbuilt map bhud is disabled with stripper.